narciuch/assignment4/test/IslandFinderTest.java

```diff
@@ -0,0 +1,33 @@
+import org.junit.Test;
+
+import static junit.framework.TestCase.assertTrue;
+
+public class IslandFinderTest {
+  IslandFinder islandFinder = new IslandFinder();
+
+  @Test
+  public void findsMultipleIslands() {
+    boolean[][] land = {{false, true, false, true}, {true, true, false, false},
+      {false, false, true, false}, {false, false, true, false}};
```

Comment:
Map would be easier to read if it was formatted as one row per line with values aligned in columns.